services/CollectionService/src/index.php

```diff
@@ -25,40 +22,25 @@
   'twig.path' => __DIR__.'/../templates',
 ));
 
-$app['twig'] = $app->share($app->extend('twig', function($twig, $app) {
-  return $twig;
-}));
 $islandoraResourceServiceProvider = new \Islandora\ResourceService\Provider\ResourceServiceProvider;
 
-$app->register($islandoraResourceServiceProvider);
+//Registers Resource Service and defines current app's path for config context
+$app->register($islandoraResourceServiceProvider, array(
+  'islandora.BasePath' => __DIR__,
+));
 $app->mount("/islandora", $islandoraResourceServiceProvider);
 $app->register(new \Islandora\ResourceService\Provider\UuidServiceProvider(), array(
   'UuidServiceProvider.default_namespace' => $app['config']['islandora']['defaultNamespaceDomainUuuidV5'],
 ));
 
-//$app['uuid'] = $app['islandora.uuid5'](rand());
+//We will use a static uuid5 to fake a real IRI to replace <> during turtle parsing
+$app['uuid5'] = $app->share(function () use ($app) {
```

Comment:
@daniel-dgi, UUIDV5 works as a constant (build from, in this case the www.islandora.ca defined the yaml file plus the route to fedora4's endpoint. But it's still an UUID so there is less Chance that people could by mistake put something like this into an input RDF that we could end stripping out and removing in our str_replace function.  I could not find a simpler way of putting there something that would work as expected. I can cache this if you want or change the IRI to urn:islandora:theuuid to make it even more failsafe.